### DISCUSSION TITLE
seeking a bootstrap freelancer developer (remote work)

### DISCUSSION THREAD

**Discussion by sungho63** [twbs/bootstrap#39043] [General]
Hi.  Where is the best to hire bootstrap dev freelancers online?
We are a startup and need one bootstrap front end developer ASAP.  (remote work) 
Our work is mostly iterating on existing front end codes with boostrap.

**Comment by strtech111** [comment]
Feel free to contact me :-)

**Reply by bhaveshjadav713** [reply]
Where can i contact to you?

**Reply by strtech111** [reply]
iamsaqlain7@gmail.com

**Reply by bhaveshjadav713** [reply]
Hello

**Reply by bhaveshjadav713** [reply]
Send email

**Comment by myplaneticket** [comment]
> Hi. Where is the best to hire bootstrap dev freelancers online?
> We are a startup and need one bootstrap front end developer ASAP. (remote work)
> Our work is mostly iterating on existing front end codes with boostrap.

hit me up, been working with Bootstrap since beta.

**Comment by sungho63** [comment]
Thank you for reaching out.
We are a startup in San Francisco Bay area.
Could you tell me more about your experience , availability and
hourly rate?   We built front end webpages  with bootstrap 5 and need
more help with iterations to perfection in the quickest time.  We need
someone
who is a bootstrap , css guru.

Sungho

**Comment by sungho63** [comment]
Please send me links to your portfolio, github, resume detailing your
programming experiences,  and hourly rate.
This is a major project and we a dev on contract basis asap.

Are you in the USA?
thanks,
Steve

**Reply by strtech111** [reply]
https://github.com/strtech111
i have developed a website recently through flask and bootstrap 
https://www.vmdownloado.com/
and i am not in usa i am asian
